config/data_sync_list/common.json

```diff
@@ -5,6 +5,20 @@
             "Description": "Host State Persisted data",
             "SyncDirection": "Active2Passive",
             "SyncType": "Immediate"
+        },
+        {
+            "Path": "/var/lib/phosphor-fan-presence/",
+            "Description": "Fan presence persisted data",
+            "SyncDirection": "Active2Passive",
+            "SyncType": "Immediate",
+            "ExcludeList": ["/var/lib/phosphor-fan-presence/control/"]
+        },
+        {
+            "Path": "/var/lib/phosphor-logging/",
+            "Description": "System logs persisted data",
+            "SyncDirection": "Bidirectional",
+            "SyncType": "Immediate",
+            "ExcludeList": ["/var/lib/phosphor-logging/extensions/pels/pelID"]
```

Comment:
Ideally, this should be moved to the OpenPOWER JSON file. However, we currently don't have support to merge configurations from multiple JSON files for the same data path. We can revisit and update this later.